contracts/02-tamagotchi-interaction/tests/interaction.rs

```diff
@@ -1,12 +1,52 @@
-use gtest::{Program, System};
-
-// TODO: 0️⃣ Copy tests from the previous lesson and push changes to the master branch
+use gtest::{Program, System, Log};
+use tamagotchi_io::*;
 
 #[test]
-fn interaction_test() {
+fn smoke_test_01() {
     let sys = System::new();
     sys.init_logger();
     let _program = Program::current(&sys);
 
-    // TODO: 6️⃣ Test new functionality
+    _program.send(2, String::from("Heriel"));
```

Comment:
Remove underscore when using the variable.
```suggestion
    program.send(2, String::from("Heriel"));
```